### PR TITLE
Fix user authorization

### PR DESCRIPTION
**Pull request by Lydias303**
So I deleted some old code that didnt allow users to view each others profiles so thats working now :)

I also fixed the navbar so the dropdown doesn't stall anymore.

And just took out that text color for links for now. 

Just some little stuff 

Already pulled down from master no conflicts. G2G

### CODE REVIEW

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -62,3 +62,5 @@ $(".to_date")
     $('#myCarousel').carousel()
   })
 }(window.jQuery)
+
+$('.dropdown-toggle').dropdown();
```
Comment (cluhring) [inline app/assets/javascripts/application.js]: don't mix double and single quotes, or else.

**Comment by cluhring** [comment]
Is host? 
hardcoded to equal true? will that cause any problems?

**Comment by cluhring** [comment]
in the seed data, I think the listings have been assigned to each of these users.  Defining their roles to a 0 might screw with some stuff.

**Comment by cluhring** [comment]
Might be why Kit was having trouble seeing some stuff?

**Comment by bryceholcomb** [comment]
Looks good!

**Pull request merged**